com.unity.render-pipelines.core/Editor/Debugging/DebugUIDrawer.Builtins.cs

```diff
@@ -865,14 +968,22 @@ internal void DisplayChild(Rect rect, DebugUI.Widget child, bool disable)
             else if (child.GetType() == typeof(DebugUI.ColorField))
             {
                 var widget = Cast<DebugUI.ColorField>(child);
-                using (new EditorGUI.DisabledScope(disable))
-                    EditorGUI.ColorField(rect, GUIContent.none, widget.GetValue(), false, widget.showAlpha, widget.hdr);
+                EditorGUI.ColorField(rect, GUIContent.none, widget.GetValue(), false, widget.showAlpha, widget.hdr);
             }
             else if (child.GetType() == typeof(DebugUI.BoolField))
             {
                 var widget = Cast<DebugUI.BoolField>(child);
-                using (new EditorGUI.DisabledScope(disable))
-                    EditorGUI.Toggle(rect, GUIContent.none, widget.GetValue());
+                EditorGUI.Toggle(rect, GUIContent.none, widget.GetValue());
+            }
+            else if (child.GetType() == typeof(DebugUI.ObjectField))
+            {
+                var widget = Cast<DebugUI.ObjectField>(child);
+                EditorGUI.ObjectField(rect, GUIContent.none, widget.GetValue(), widget.type, true);
+            }
+            else if (child.GetType() == typeof(DebugUI.ObjectListField))
+            {
+                var widget = Cast<DebugUI.ObjectListField>(child);
+                DebugUIDrawerObjectListField.DoObjectList(rect, widget, widget.GetValue());
```

Comment:
we should use the appropriate DebugUIDrawers here but the rely on GUILayout, and therefore don't work here wher we want to give a specific rect.
I tried to use BeginArea / EndArea but it doesn't work, maybe it should be reinvestigated at some point

Reply:
Do not bother with this as with @arttu-peltonen  we will try to move the debug UI to UTK, I think is good for now